### ISSUE TITLE
Brow.sh demo seems to be down?

### ISSUE THREAD

**Issue by 2fst4u** [browsh-org/browsh#321]
Not so much a bug report, and I'm sure you're aware of this but the demo seems to not be working. I couldn't see any other mention of this in the issues.

**Comment by tombh** [comment]
Yes :( There were some issues I couldn't figure out, and I don't have time to fix them :'( So I just had to turn them off for now. Sorry.

**Comment by filips123** [comment]
Do you know when it will work again?

**Comment by gardar** [comment]
> Yes :( There were some issues I couldn't figure out, and I don't have time to fix them :'( So I just had to turn them off for now. Sorry.

What kind of issues? Anything we could do to help get it going again?

**Comment by tombh** [comment]
Thanks @gardar! For some reason both the SSH service and HTML service were never starting up, I hadn't changed any code in Browsh to explain it, and I reinstalled the cluster, but still I couldn't see the problem. Maybe it was just temporary. But I just don't have so much time these days cos of my job.

Anyway here's the cluster code if you're curious: https://github.com/browsh-org/baas

**Comment by niutech** [comment]
The demo servers is still down. Where can I find another Browsh demo server?
